### DISCUSSION TITLE
Version 1.15.2

### DISCUSSION THREAD

**Discussion by shazzaam7** [xenia-manager/xenia-manager#182] [General]
## What's Changed
* Fixed installation of content for Xenia Canary
* Disabled XboxUnity source for downloading title updates per request
* Improved Xbox Marketplace source for adding games where there was an issue with mismatched game title info (Example: Ninja Gaiden 3 and Ninja Gaiden 3 Razor's Edge). Now games should be named properly

**Full Changelog**: https://github.com/xenia-manager/xenia-manager/compare/1.15.1...1.15.2

<hr /><em>This discussion was created from the release <a href='https://github.com/xenia-manager/xenia-manager/releases/tag/1.15.2'>Version 1.15.2</a>.</em>

**Comment by gameriom** [comment]
You guys could make Xenia run the Xbox InGame Menu(that one that appears after pressing the Xbox Logo on the controller), i know that this is Xenia Manager and not Xenia but it would be interesting, maybe as an overlay
If this is going to be made please make look exactly as in the 360, without modernizing it

**Reply by gameriom** [reply]
And it could do some cool functions too like music, open other games, configure profiles and etc.

**Comment by MPUltraX** [comment]
> Disabled XboxUnity source for downloading title updates per request

Why?

**Reply by shazzaam7** [reply]
Got requested to take it down since it's considered piracy.

**Comment by Ern-Ani** [comment]
There is something wrong with this version, it is detected as infected and deleted by the antivirus when downloading without even opening it, the previous versions and the pre versions this does not happen.

**Reply by shazzaam7** [reply]
Probably a false positive by Windows Defender. Nothing I can really do about it.

**Reply by Boykalord** [reply]
VirusTotal gives like 10 warnings, it's not just WD.
Was going to give it a try but here is something wrong here.

**Reply by shazzaam7** [reply]
All of the source code is available so you can compile it yourself.
Also a note: v2 prereleases require to be run as administrator to work and they return less "positives" than v1 that doesn't require to be run as administrator.

**Reply by Boykalord** [reply]
I am not accusing you of anything, just to be clear.
just letting you know that its not just WD being picky.
I'll try the next version, cheers.

**Reply by shazzaam7** [reply]
Oh no, I'm just pointing it out in case someone else comes out with the VirusTotal results so I don't have to repeat myself multiple times 😄

**Comment by D3V4NN** [comment]
hi guys, i don't know what's wrong but my tenchu Z game is all buggy now.

**Reply by D3V4NN** [reply]
Hi guys, I'm having problems with glitz graphics in the game Tenchu Z.

**Reply by D3V4NN** [reply]
I managed to solve the problem by activating the option to clear the GPU cache.
